modules/deploy-red-hat-openshift-container-storage.adoc

```diff
@@ -9,57 +9,57 @@
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/managing_and_allocating_storage_resources[Managing and allocating resources]
 
 |Managing storage resources across a hybrid cloud or multicloud environment using the Multicloud Object Gateway (NooBaa)
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/managing_hybrid_and_multicloud_resources[Managing hybrid and multicloud resources]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/managing_hybrid_and_multicloud_resources[Managing hybrid and multicloud resources]
 
 |Safely replacing storage devices for {rh-storage-first}
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/replacing_devices[Replacing devices]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/replacing_devices[Replacing devices]
 
 |Safely replacing a node in a {rh-storage-first} cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/replacing_nodes[Replacing nodes]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/replacing_nodes[Replacing nodes]
 
 |Scaling operations in {rh-storage-first}
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/scaling_storage[Scaling storage]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/scaling_storage[Scaling storage]
 
-|Monitoring a {rh-storage-first} 4.12 cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/monitoring_openshift_data_foundation[Monitoring Red Hat OpenShift Data Foundation 4.12]
+|Monitoring a {rh-storage-first} 4.13 cluster
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/monitoring_openshift_data_foundation[Monitoring Red Hat OpenShift Data Foundation 4.13]
 
 |Resolve issues encountered during operations
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.12/html/troubleshooting_openshift_data_foundation[Troubleshooting OpenShift Data Foundation 4.12]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.13/html/troubleshooting_openshift_data_foundation[Troubleshooting OpenShift Data Foundation 4.13]
 
 |Migrating your {product-title} cluster from version 3 to version 4
-|link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/migrating_from_version_3_to_4/index[Migration]
+|link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/migrating_from_version_3_to_4/index[Migration]
```

Comment:
```suggestion
|link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html/migrating_from_version_3_to_4/index[Migration from version 3 to 4]
```
Minor edit to match the title of the page you're linking to.

Reply:
Afraid this will be forgotten with each new release, so leaving it as is.